catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -259,32 +269,26 @@ protected Object executeWithSubject() throws Exception {
           }
         } catch (InvalidSyntaxException e) {
           LOGGER.info("No Zip Transformer found.  Unable export metacards to a zip file.");
+        } catch (CatalogTransformerException e) {
+          LOGGER.info("zipCompression transform failed");
         }
       } else if (multithreaded > 1) {
         final List<Result> results = new ArrayList<>(response.getResults());
         executorService.submit(
             () -> {
-              boolean transformationFailed = false;
               for (final Result result : results) {
                 Metacard metacard = result.getMetacard();
                 try {
-                  exportMetacard(dumpDir, metacard);
-                  printStatus(resultCount.incrementAndGet());
-                } catch (IOException | CatalogTransformerException e) {
-                  transformationFailed = true;
+                  exportMetacard(dumpDir, metacard, resultCount);
+                } catch (IOException e) {
                   LOGGER.debug("Failed to dump metacard {}", metacard.getId(), e);
                 }
               }
-              if (transformationFailed) {
-                LOGGER.info(
-                    "One or more metacards failed to transform. Enable debug log for more details.");
-              }
             });
-      } else {
+      } else if (multithreaded == 1) {
```

Comment:
✏️ Why was this check added here? Are there unit tests for when multithreaded = 1 and multithreaded < 1?

Reply:
This was not supposed to be in here. It has been removed now.